authentication-typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/react-native-template-authentication-typescript",
-    "version": "3.0.0",
+    "version": "3.0.0-alpha.2",
```

Comment:
you need to update the alpha version for blank and routing templates as well.

Reply:
the only change is for authentication template
no need for blank and routing templates